### PR TITLE
docs: guide to capabilities in multi-windows app

### PR DESCRIPTION
**Pull request by vdang-crabnebula**
#### Description

- What does this PR change? Give us a brief description. 

This introduces a new step-by-step guide to use capabilities in a Tauri app with multiple windows.

### CODE REVIEW

File: src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+    #### Give filesystem capabilities to the `first` window
+
+    We give the `first` window the capability to have read access to the content of the `$HOME` directory.
+
+    <ShowSolution>
+    Use the `windows` field in a capability file with one or multiple window labels.
+
+    *filesystem.json:*
+    ```json
```
Comment (FabianLars) [inline src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx]:
```suggestion
    ```json title="filesystem.json"
```

File: src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    #### Give dialog capabilities to the `first` and `second` window
+
+    We give to `first` and `second` windows the capability to create a "Yes/No" dialog
+
+    <ShowSolution>
+    Use the `windows` field in a capability file with one or multiple window labels.
+
+    *dialog.json:*
+
+    ```json
```
Comment (FabianLars) [inline src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx]:
```suggestion
    ```json title="dialog.json"
```

File: src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+3. ### Make Capabilities Platform Dependent
+
+    We now want to customize the capabilities to be active only on certain platforms.
+    We make our filesystem capabilities only active on `linux` and `windows`.
+
+    <ShowSolution>
+    Use the `platforms` field in a capability file to make it platform-specific.
+
+    *filesystem.json:*
+    ```json
```
Comment (FabianLars) [inline src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx]:
```suggestion
    ```json title="filesystem.json"
```

File: src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+    {
+      "identifier": "fs-read-home",
+      "description": "Allow access file access to home directory",
+      "local": true,
+      "windows": ["first"],
+      "permissions": [
+        "fs:allow-home-read",
+      ]
```
Comment (FabianLars) [inline src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx]:
```suggestion
      ],
```

File: src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx [hunk 5]
```diff
@@ -0,0 +1,191 @@
+---
+title: Capabilities for Different Windows and Platforms
+sidebar:
+  order: 1
```
Comment (FabianLars) [inline src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx]:
```suggestion
  order: 11
```

File: src/content/docs/learn/Security/using-plugin-permissions.mdx [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Using Plugin Permissions
 sidebar:
+  order: 0
```
Comment (FabianLars) [inline src/content/docs/learn/Security/using-plugin-permissions.mdx]:
```suggestion
  order: 10
```

File: src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+    </ShowSolution>
+
+</Steps>
+
+## Conclusion and Resources
+
+We have learned how to create multiple windows in a Tauri app and give them specific capabilities. Furthermore these capabilities can also be targeted to certain platforms.
+
+An example application that used window capabilities can be found in the [`api` example](https://github.com/tauri-apps/tauri/tree/dev/examples/api) of the [Tauri Github repository](https://github.com/tauri-apps/tauri).
+The fields that can be used in a capability file are listed in the [References/Security/Capability section](reference/acl/capability).
```
Comment (FabianLars) [inline src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx]:
```suggestion
The fields that can be used in a capability file are listed in the [Capability](/reference/acl/capability) reference.
```

**Review by FabianLars — APPROVED** [review]
Thanks :)

File: src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx [hunk 8]
```diff
@@ -0,0 +1,187 @@
+---
+title: Capabilities for Different Windows and Platforms
+sidebar:
+  order: 11
+  badge:
+    text: WIP
```
Comment (dreyfus92) [inline src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx]: @FabianLars, do we need this wip badge?

Comment (FabianLars) [reply src/content/docs/learn/Security/capabilities-for-windows-and-platforms.mdx]: nope, good call

**Pull request merged**